ui/src/app/workflows/components/workflow-details/workflow-details.tsx

```diff
@@ -47,6 +47,10 @@ export class WorkflowDetails extends React.Component<RouteComponentProps<any>, W
     private changesSubscription: Subscription;
     private timelineComponent: WorkflowTimeline;
 
+    private get resourceVersion() {
+        return this.state.workflow && this.state.workflow.metadata.resourceVersion;
```

Comment:
If `this.state.workflow.metadata === null`, then we return the whole workflow object. Perhaps this should be in an `if` check as opposed to inline?

Reply:
I don't think `this.state.workflow.metadata` could every be null?

Reply:
Even so, it would probably be good for our code to be tolerant

Reply:
Up to you